next.config.js

```diff
@@ -25,7 +25,7 @@ module.exports = {
   sassOptions: {
     includePaths: [path.join(__dirname, "styles")],
   },
-  reactStrictMode: true,
+  reactStrictMode: false,
```

Comment:
Any specific reason to turn this off?

Reply:
Only when I was testing in React development mode!  Thanks didn't mean to include that in the final code changes.